Makefile

```diff
@@ -1,8 +1,9 @@
-.PHONY: build 
+.PHONY: build
 build:
 	@tools/bazel build \
 		--disk_cache=$(HOME)/Library/Caches/Bazel \
-		--spawn_strategy=standalone \
+		--spawn_strategy=local \
+		--use_top_level_targets_for_symlinks \
```

Comment:
@dgcoffman is this now required for the non header map build?

Reply:
Sorry, I didn't understand the question. What's the "non header map build"?

I added `--use_top_level_targets_for_symlinks` here because something (I'm not sure what but I think the updated rules_apple?) causes an additional bindir to be created, which bazel-bin then symlinks to instead of the one which actually contains RepoTools and Compiler.  `use_top_level_targets_for_symlinks` just makes bazel-bin point at the directory it did previously.